source/operator.rst

```diff
@@ -218,6 +218,60 @@ configuration options for the Percona XtraDB Cluster.
+|                 | .. _pxc-expose-loadbalancersourceranges:                                                  |
+|                 |                                                                                           |
+| **Key**         | `pxc.expose.loadBalancerSourceRanges                                                      |
+|                 | <operator.html#pxc-expose-loadbalancersourceranges>`_                                     |
++-----------------+-------------------------------------------------------------------------------------------+
+| **Value Type**  | string                                                                                    |
++-----------------+-------------------------------------------------------------------------------------------+
+| **Example**     | ``10.0.0.0/8``                                                                            |
++-----------------+-------------------------------------------------------------------------------------------+
+| **Description** | The range of client IP addresses from which the load balancer should be reachable         |
+|                 | (if not set, there is no limitations)                                                     |
++-----------------+-------------------------------------------------------------------------------------------+
+|                                                                                                             |
++-----------------+-------------------------------------------------------------------------------------------+
+|                 | .. _pxc-expose-annotations:                                                               |
+|                 |                                                                                           |
+| **Key**         | `pxc.expose.annotations.networking.gke.io/load-balancer-type                              |
+|                 | <operator.html#pxc-expose-annotations>`_                                                  |
++-----------------+-------------------------------------------------------------------------------------------+
+| **Value Type**  | string                                                                                    |
++-----------------+-------------------------------------------------------------------------------------------+
+| **Example**     | ``Internal``                                                                              |
++-----------------+-------------------------------------------------------------------------------------------+
+| **Description** |                                                                                           |
+|                 |                                                                                           |
+|                 |                                                                                           |
```

Comment:
Out of words? :)

Attach metadata to the Service object through the `annotation https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/`_. ?